src/components/Login/index.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { Box, Image } from '@chakra-ui/core';
+import { Redirect } from 'react-router-dom';
+import { useLoginSelector, useLogintDispatch } from '../../hooks/login';
+import './style.css';
+import logo from '../../static/codelibrarylogo.png';
+
+export default function Login() {
+    const { accessToken } = useLoginSelector();
+    const { signInUser } = useLogintDispatch();
+
+    const handleClick = () => {
+        signInUser();
+    };
```

Comment:
The Login component should accept `handleClick` function and the Login component should only be rendered on the Login page if the access token is available